src/components/Articles.jsx

```diff
@@ -0,0 +1,40 @@
+import { useEffect, useState } from "react";
+import { getArticles } from "../api";
+import ArticleCard from "./ArticleCard";
+
+const Articles = () => {
+    const [articles, setArticles] = useState([]);
+
+    useEffect(() => {
+        getArticles().then((res) => {
+            setArticles(res.articles)
+        })
+    })
+
+    return (
+        <>
+            <h1>Articles</h1>
+            <ul className="article-list">
+            {articles.map((article) => {
+                return (
+                    <li>
+                        <ArticleCard>
+                        <>
+                            <img id= 'article-image' src={article.article_img_url}/>
+                        </>
+                        <div className="article-text">
+                            <h3>{article.title}</h3>
+                            <p><b>Author:</b> {article.author}</p>
+                            <p id="topic"><b>Topic:</b> {article.topic}</p>
+                            <p><b>Date Posted: </b>{article.created_at}</p>
+                        </div>
+                        </ArticleCard>
```

Comment:
How you are styling the `ArticleCard` component determines whether passing the html tags as children is necessary. For example, if the card can take any children and style them regardless of what they are neatly into the card, then passing children makes sense. However, if it can only effectively style particular html in a particular order then it would make more sense to have the html extracted to the component. It comes down to whether this component is reusable else where in your code and whether you need the ability to pass different children